test/helpers/cmd.go

```diff
@@ -205,6 +205,41 @@ func (res *CmdRes) Filter(filter string) (*FilterBuffer, error) {
 	return &FilterBuffer{result}, nil
 }
 
+// FilterLines works like Filter, but applies the JSONPath filter to each line
+// separately and returns returns a FilterBuffer for each line. An error is
+// returned only for the first line which cannot be unmarshalled.
+func (res *CmdRes) FilterLines(filter string) ([]FilterBuffer, error) {
```

Comment:
Is this used anywhere?

Reply:
Not yet. However, I expect this to be used in the Hubble tests we will be adding in the upcoming days. If you feel this addition is premature, I can remove it again.